demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java

```diff
@@ -269,6 +280,10 @@ public void applyDefinitions(Configuration config) {
 	}
 
 	public static void main(String[] args) {
+		
+		db = new HashMapCtxDB();
```

Comment:
Please move that to [PlugtestServer.init(BaseConfig)](https://github.com/eclipse/californium/blob/master/demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java#L351) and try to use arguments for `db` and `serverRid` instead of `static` fields.

Reply:
I tried to restructure things accordingly in commit c9dff1a0e4c982531367ca05e9c287a174bf7f48